docs/backup/08-02-restore-backup.md

```diff
@@ -50,3 +52,27 @@ Follow these steps to restore resources:
     >```bash
     > kubectl get virtualservices --all-namespaces
     > ```
+
+5. Once the restore succeeds, remove the `velero` namespace:
+
+    ```bash
+    kubectl delete ns velero
+    ```
+
+## Troubleshooting
```

Comment:
Can we extract this part to the Troubleshooting guide? We have a new type of doc, called troubleshooting: https://github.com/kyma-project/community/blob/master/guidelines/templates/resources/troubleshooting.md.
So we would have 2 small troubleshooting docs:
10-01-restore-troubleshooting